lam4doc/entrypoints/ui/helpers.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+
+# helpers.py
+# Date:  06/01/2021
+# Author: Mihai Coșleț
+# Email: coslet.mihai@gmail.com 
+
+"""
+Misfit methods
+"""
+from json import loads
+
+
+def get_error_message_from_response(response):
```

Comment:
looks good, a consistent way of handling errors; I would however not use an entire module for a single function. perhaps this could live in the __init__.py (in the worst case, if not in the module where it is used)